Cargo.toml

```diff
@@ -10,6 +10,33 @@ members = [
   "cosmic-comp-config",
 ]
 
+[workspace.dependencies]
+tracing = { version = "0.1.37", features = ["max_level_debug", "release_max_level_info"] }
+xkbcommon = "0.7"
+
+[workspace.dependencies.smithay]
+default-features = false
+features = [
+  "backend_drm",
+  "backend_gbm",
+  "backend_egl",
+  "backend_libinput",
+  "backend_session_libseat",
+  "backend_udev",
+  "backend_winit",
+  "backend_vulkan",
+  "backend_x11",
+  "desktop",
+  "use_system_lib",
+  "renderer_glow",
+  "renderer_multi",
+  "wayland_frontend",
+  "xwayland",
+]
+git = "https://github.com/smithay/smithay.git"
+rev = "74ef59a3f"
+version = "0.3"
```

Comment:
See https://github.com/pop-os/cosmic-settings-daemon/pull/29#pullrequestreview-2131064839. I would be very cautious to make this a workspace dependency and would very much prefer to avoid that.